kuma/static/styles/includes/_svg_data_uri_icons.scss

```diff
@@ -0,0 +1,13 @@
+$icon-white-clock: 'data:image/svg+xml;base64,PHN2ZyBzdHlsZT0iZmlsbDogI2ZmZjsiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI4IiB2aWV3Qm94PSIwIDAgMjQgMjgiPjxwYXRoIGQ9Ik0xNCA4LjV2N2MwIC4yODEtLjIxOS41LS41LjVoLTVhLjQ5NC40OTQgMCAwIDEtLjUtLjV2LTFjMC0uMjgxLjIxOS0uNS41LS41SDEyVjguNWMwLS4yODEuMjE5LS41LjUtLjVoMWMuMjgxIDAgLjUuMjE5LjUuNXptNi41IDUuNWMwLTQuNjg4LTMuODEzLTguNS04LjUtOC41UzMuNSA5LjMxMyAzLjUgMTRzMy44MTMgOC41IDguNSA4LjUgOC41LTMuODEzIDguNS04LjV6bTMuNSAwYzAgNi42MjUtNS4zNzUgMTItMTIgMTJTMCAyMC42MjUgMCAxNCA1LjM3NSAyIDEyIDJzMTIgNS4zNzUgMTIgMTJ6Ii8+PC9zdmc+';
+
+$icon-green-smile: 'data:image/svg+xml;base64,PHN2ZyBzdHlsZT0iZmlsbDogIzRkOWYwYzsiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI4Ij48cGF0aCBkPSJNMTcuNzE5IDE2Ljc5N0MxNi45MzggMTkuMzEzIDE0LjY0MSAyMSAxMiAyMXMtNC45MzctMS42ODctNS43MTktNC4yMDNjLS4xNzItLjUzMS4xMjUtMS4wNzguNjU2LTEuMjUuNTE2LS4xNzIgMS4wNzguMTI1IDEuMjUuNjU2QzguNzAzIDE3Ljg3NSAxMC4yNSAxOSAxMiAxOXMzLjI5Ny0xLjEyNSAzLjgxMy0yLjc5N2ExLjAwNiAxLjAwNiAwIDAgMSAxLjI2Ni0uNjU2Ljk5NC45OTQgMCAwIDEgLjY0MSAxLjI1ek0xMCAxMGMwIDEuMTA5LS44OTEgMi0yIDJzLTItLjg5MS0yLTIgLjg5MS0yIDItMiAyIC44OTEgMiAyem04IDBjMCAxLjEwOS0uODkxIDItMiAycy0yLS44OTEtMi0yIC44OTEtMiAyLTIgMiAuODkxIDIgMnptNCA0YzAtNS41MTYtNC40ODQtMTAtMTAtMTBTMiA4LjQ4NCAyIDE0czQuNDg0IDEwIDEwIDEwIDEwLTQuNDg0IDEwLTEwem0yIDBjMCA2LjYyNS01LjM3NSAxMi0xMiAxMlMwIDIwLjYyNSAwIDE0IDUuMzc1IDIgMTIgMnMxMiA1LjM3NSAxMiAxMnoiLz48L3N2Zz4=';
+
+$icon-red-frown: 'data:image/svg+xml;base64,  PHN2ZyBzdHlsZT0iZmlsbDogI2U2NjQ2NTsiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI4Ij48cGF0aCBkPSJNMTcuNzE5IDE5LjIwM2EuOTkzLjk5MyAwIDAgMS0uNjQxIDEuMjUgMS4wMDcgMS4wMDcgMCAwIDEtMS4yNjYtLjY1NkMxNS4yOTYgMTguMTI1IDEzLjc0OSAxNyAxMS45OTkgMTdzLTMuMjk3IDEuMTI1LTMuODEzIDIuNzk3Yy0uMTcyLjUzMS0uNzM0LjgyOC0xLjI1LjY1Ni0uNTMxLS4xNzItLjgyOC0uNzE5LS42NTYtMS4yNUM3LjA2MSAxNi42ODcgOS4zNTggMTUgMTEuOTk5IDE1czQuOTM3IDEuNjg3IDUuNzE5IDQuMjAzek0xMCAxMGMwIDEuMTA5LS44OTEgMi0yIDJzLTItLjg5MS0yLTIgLjg5MS0yIDItMiAyIC44OTEgMiAyem04IDBjMCAxLjEwOS0uODkxIDItMiAycy0yLS44OTEtMi0yIC44OTEtMiAyLTIgMiAuODkxIDIgMnptNCA0YzAtNS41MTYtNC40ODQtMTAtMTAtMTBTMiA4LjQ4NCAyIDE0czQuNDg0IDEwIDEwIDEwIDEwLTQuNDg0IDEwLTEwem0yIDBjMCA2LjYyNS01LjM3NSAxMi0xMiAxMlMwIDIwLjYyNSAwIDE0IDUuMzc1IDIgMTIgMnMxMiA1LjM3NSAxMiAxMnoiLz48L3N2Zz4=';
```

Comment:
nit: inconsistent spacing (I think it's meant to align the PHNs but it falls apart pretty quickly)